eng/tools/rush-runner/src/helpers.js

```diff
@@ -18,19 +18,16 @@ export const reducedDependencyTestMatrix = {
     "@azure/template",
   ],
   "test-utils": [
```

Comment:
skipping storage-blob until https://github.com/Azure/azure-sdk-for-js/pull/33329 is merged.

storage-blob depends on v3 recorder pulling older core-rest-pipeline which is incompatible with the current versions in the repo.

cc @jeremymeng